ui/00_message/achievement/achievement_info.xml

```diff
@@ -6457,21 +6467,22 @@ level 30 in crafting</translation>
   <message>
     <key>ACHIEVEMENT_INFO_2362</key>
     <original>タラスクを1体討伐</original>
-    <translation>Kill 1 タラスクを</translation>
+    <translation>Kill 1 Tarasque</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2363</key>
     <original>タラスクを10体討伐</original>
-    <translation>Kill 10 タラスクを</translation>
+    <translation>Kill 10 Tarasque</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2364</key>
     <original>タラスクを100体討伐</original>
-    <translation>Kill 100 タラスクを</translation>
+    <translation>Kill 100 Tarasque</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2368</key>
     <original>グリーンガーディアンを50体討伐</original>
+    <translation>Kill 50 グリーンガーディアンを</translation>
```

Comment:
This is `Green Guardian`.

I will probably add a replication config in the future to catch kill quests like this.